pkg/services/alerting/notifier.go

```diff
@@ -219,7 +219,7 @@ func (n *notificationService) renderAndUploadImage(evalCtx *EvalContext, timeout
 		},
 		Width:           1000,
 		Height:          500,
-		ConcurrentLimit: n.cfg.AlertingRenderLimit,
+		ConcurrentLimit: 0,
```

Comment:
This is just to satisfy the compiler for now – when we remove `pkg/services/alerting` this should no longer be of any concern.